### PR TITLE
fix: rename zk chains

### PR DESCRIPTION
**Pull request by zk-Lumi**
# What :computer: 
* Renamed all references to ZK Chain

# Why :hand:
* New guidance

### CODE REVIEW

File: docs/zk-stack/running-a-zk-chain/getting-started.md
Comment (MexicanAce) [inline docs/zk-stack/running-a-zk-chain/getting-started.md:1]: This file is empty, we should migrate the content

Comment (zk-Lumi) [reply docs/zk-stack/running-a-zk-chain/getting-started.md:1]: @MexicanAce this page was already empty on [live site](https://docs.zksync.io/zk-stack/running-a-hyperchain/getting-started.html) let me know if I should delete the page

Comment (MexicanAce) [reply docs/zk-stack/running-a-zk-chain/getting-started.md:1]: ah, yes plz. Let's delete

Comment (zk-Lumi) [reply docs/zk-stack/running-a-zk-chain/getting-started.md:1]: Done ✅

**Review by MexicanAce — APPROVED** [review]
LGTM

**Pull request merged**